Atomic/syscontainers.py

```diff
@@ -549,17 +548,14 @@ def update_system_container(self, name):
         self._checkout_system_container(repo, name, image, next_deployment, True, values, remote=self.args.remote)
 
     def get_container_runtime_info(self, container):
-        if self.user:
-            return {'status' : "running" if self._is_service_active(container) else "exited"}
 
-        try:
-            inspect_stdout = util.check_output([util.RUNC_PATH, "state", container], stderr=DEVNULL)
-            ret = json.loads(inspect_stdout.decode())
-            status = ret["status"]
-            created = dateparse(ret['created'])
```

Comment:
Currently we display the "created time" of the container saved in info, which is when we installed the container. I think that is better because if we systemctl stop, and then systemctl start, the runc created time would be refreshed, and it would display LAST run-time instead.